lib/vbox/vbox.go

```diff
@@ -266,6 +294,6 @@ func (self *VboxConfig) GetCpu() int {
 	return int(self.Option.Cpu)
 }
 
-func (self *VboxConfig) GetUsbs() (string, string, string) {
+func (self *VboxConfig) GetUsbs() (usb, ehci, xhci OnOff) {
```

Comment:
__[golint]__
* [golint] exported method VboxConfig.GetUsbs should have comment or be unexported
* [golint] receiver name should be a reflection of its identity; don't use generic names such as "this" or "self"

<!-- Comment by [SideCI](https://sideci.com) -->